src/dotnet-xunit/Program.cs

```diff
@@ -229,11 +229,19 @@ static void PrintUsage()
         Console.WriteLine("  -method \"name\"         : run a given test method (can be fully specified or use a wildcard;");
         Console.WriteLine("                         : i.e., 'MyNamespace.MyClass.MyTestMethod' or '*.MyTestMethod')");
         Console.WriteLine("                         : if specified more than once, acts as an OR operation");
+        Console.WriteLine("  -nomethod \"name\"       : do not run a given test method (can be fully specified or use a wildcard;");
+        Console.WriteLine("                         : i.e., 'MyNamespace.MyClass.MyTestMethod' or '*.MyTestMethod')");
+        Console.WriteLine("                         : if specified more than once, acts as an AND operation");
         Console.WriteLine("  -class \"name\"          : run all methods in a given test class (should be fully");
         Console.WriteLine("                         : specified; i.e., 'MyNamespace.MyClass')");
         Console.WriteLine("                         : if specified more than once, acts as an OR operation");
+        Console.WriteLine("  -noclass \"name\"        : do not run any methods in a given test class (should be fully");
+        Console.WriteLine("                         : specified; i.e., 'MyNamespace.MyClass')");
+        Console.WriteLine("                         : if specified more than once, acts as an AND operation");
         Console.WriteLine("  -namespace \"name\"      : run all methods in a given namespace (i.e.,");
         Console.WriteLine("                         : 'MyNamespace.MySubNamespace')");
+        Console.WriteLine("  -nonamespace \"name\"    : do not run any methods in a given namespace (i.e.,");
```

Comment:
L245 `Console.WriteLine("                         : if specified more than once, acts as an OR operation");` belongs to the command above "-namespace". If the OR also applies to "-nonamespace" you should probably copy and paste that line above.

Reply:
Good catch, thanks!